src/custom/pages/CowGame/index.tsx

```diff
@@ -30,6 +30,10 @@ const Wrapper = styled(Page)`
 `
 
 export default function CowGamePage() {
+  if (0 === 0) {
```

Comment:
????

Reply:
will be reverted, its too force an error. Briefly commented in the test description